uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1770,72 +1785,84 @@ or a phantom
 
 sub validateCandidate {
     my $this = shift;
-    my ($subjectIDsref)= @_;
+    my ($subjectIDsref) = @_;
+
     my $CandMismatchError = undef;
+
+
+    ############################################################
+    ################ No Checking if the subject is Phantom #####
+    ############################################################
+    if ($subjectIDsref->{'isPhantom'}) {
```

Comment:
moved this up in the function as there is not need to do anything else if it is a phantom.